src/client/src/shell/notification/TradeNotification.stories.tsx

```diff
@@ -1,10 +1,11 @@
 import { action } from '@storybook/addon-actions'
-import { select } from '@storybook/addon-knobs'
+import { capitalize } from 'lodash'
 import { storiesOf } from '@storybook/react'
 import React from 'react'
 import { Story } from 'rt-storybook'
 import { styled } from 'rt-theme'
 import TradeNotification, { Props } from './TradeNotification'
+import {TradeStatus} from 'rt-types'
```

Comment:
Minor: Formatting `{ Trade Status }`

Reply:
I don't understand your comment, TradeStatus has been formatted correctly and it is a type.